packages/packages/site-settings/src/components/primary-action.tsx

```diff
@@ -0,0 +1,36 @@
+import { useActiveDocument, useActiveDocumentActions } from '@elementor/documents';
+import { Button, CircularProgress, Box } from '@elementor/ui';
+import { __ } from '@wordpress/i18n';
+
+export default function PrimaryAction() {
+	const document = useActiveDocument();
+	const { save } = useActiveDocumentActions();
+
+	return (
+		<Box sx={ {
+			position: 'absolute',
+			bottom: 0,
+			right: 0,
+			left: 0,
+			px: 5,
+			py: 4,
+			borderTop: 1,
+			borderColor: 'grey.200',
+			background: 'white',
+		} }>
+			<Button
+				variant="contained"
+				disabled={ ! document || ! document.isDirty }
```

Comment:
maybe don't render at all when there is no document?

Reply:
No